charts/api7/templates/api-usage-service.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if (.Values.api_usage.configuration | default dict).enable }}
```

Comment:
```suggestion
{{- if (.Values.api_usage| default dict).enable }}
```

Reply:
please try to deploy this helm chart in your local cluster to confirm all template is working.

Reply:
Resoved it.
I `helm install`ed this chart, and all services are OK.
![image](https://github.com/user-attachments/assets/4187a659-d6bd-4068-a14f-579f46a7b557)

![image](https://github.com/user-attachments/assets/178e25cd-7a29-4730-98bc-8e8b3a429555)

Reply:
good job